index.js

```diff
@@ -1,81 +1,117 @@
  @Example - Sending the function a value of 5 will result in 5 "Echo!"'s in the terminal
  @Test - Make a function call of  `makeEchoes(5)`
 */
-
-/* 
- @Challenge 03 - Write a function named `showMeTheMoney` that loops from 1 to 200. 
+makeEchoes = n => {
+    while (n) {
+        console.log("Echo")
+        n--
+    }
+}
+makeEchoes(5)
+/*
+ @Challenge 03 - Write a function named `showMeTheMoney` that loops from 1 to 200.
  Use a condition in the loop to console.log the number when it is over 190
  @Example - Expect to see 191, 192, 193, 194, 195, 196, 197, 198, 199, 200 in the terminal
  @Test - Make a function call of  `showMeTheMoney()`
 */
-
-/* 
- @Challenge 04 - Write a function named `showTheFifties` that loops from 1 to 200. 
+showMeTheMoney = () => {
+    for (let index = 0; index < 201; index++) {
+        if (index > 190) {
+            console.log(index)
+        }
+    }
+}
+showMeTheMoney()
+/*
+ @Challenge 04 - Write a function named `showTheFifties` that loops from 1 to 200.
  Use a condition in the loop to console.log the number when it is over 49 and under 60
  @Example - Expect to see 50, 51, 52, 53, 54, 55, 56, 57, 58, 59 in the terminal
  @Test - Make a function call of  `showTheFifties()`
 */
-
-/* 
+showTheFifties = () => {
+    for (let index = 0; index < 201; index++) {
+        if (index > 49 && index < 60) {
+            console.log(index)
+        }
+    }
+}
+showTheFifties()
+/*
  @Challenge 05 - Write a function that returns an array that is named `getTheTwenties` that loops from 1 to 60 and returns an array of all the 20's
  Use a condition in the loop to push to an array when the number is greater than or equal to 20 and under 30
  @Example - Expect an array returned with value of [20, 21, 22, 23, 24, 25, 26, 27, 28, 29]
  @Test - console.log the result of a function call of  `getTheTwenties()` and expect to see an array value of [20, 21, 22, 23, 24, 25, 26, 27, 28, 29]
 */
-
-/* 
+getTheTwenties = () => {
+    let returnArray = []
+    for (let index = 0; index < 61; index++) {
+        if (index >= 20 && index < 30) {
+            returnArray.push(index)
+        }
+    }
+    console.log(returnArray)
+}
+getTheTwenties()
+/*
  @Challenge 06 - Write a function that accepts an array names. Name the function `sayHiToJim` that loops through the array of names.
  Use a condition in the loop to check if the name is "Jim". If the name is "Jim" console.log a message to say "Hi Jim"
  Use conditional logic to console.log "Ignoring " and then the name of the person being ignored.
  @Example - Expect to see "Ignorning Jane" and "Ignoring Anita Bath" when the function is given an array of ["Jane", "Anita Bath"]
  @Test01 - Make a function call `sayHiToJim(["Jane", "Anita Bath"])`
  @Test02 - Make a function call `sayHiToJim(["Jane", "Anita Bath", "Jim", "Sam Sung"])`
 */
+sayHiToJim = (names) => {
```

Comment:
Looks good. Might be missing code for: "Expect to see "Ignorning Jane" and "Ignoring Anita Bath" when the function is given an array of ["Jane", "Anita Bath"]"